RNA-Seq/RNA-Seq-hands-on.md

```diff
@@ -44,7 +44,7 @@ This exercise uses RNA-seq data from the study by [Brooks et al. 2011](http://ge
 
 - Annoying but necessary: Before continuing with the mapping step, change the file type of the FASTQ files to *fastqsanger*. This signals that the FASTQ contains Sanger-scaled quality values. **TopHat2** will otherwise not be able to accept the files for input! This can be done by clicking on the pencil item of the dataset (edit attributes) →  Datatype →  select fastqsanger
 
-- If you want **TopHat** to take advantage from already known reference gene annotations, make sure that the data is available in your current Galaxy history. For this exercise, please import the Ensembl gene annotation for [Drosophila_melanogaster.BDGP5.78.gtf] (https://drive.google.com/file/d/0B9urRnOAUUI8cE1YM0k3WmJ3Njg/view?usp=sharing) by right-clicking → copy link location and paste the link in Galaxy → Upload File from your computer → paste/fetch data → start.
+- If you want **TopHat** to take advantage from already known reference gene annotations, make sure that the data is available in your current Galaxy history. For this exercise, please import the Ensembl gene annotation for [Drosophila_melanogaster.BDGP5.78.gtf](https://drive.google.com/file/d/0B9urRnOAUUI8cE1YM0k3WmJ3Njg/view?usp=sharing) by right-clicking → copy link location and paste the link in Galaxy → Upload File from your computer → paste/fetch data → start.

```

Comment:
@bgruening Of course. We already talked about that with @erxleben. We planned to do that next week